objectbox/lib/src/relations/to_many.dart

```diff
@@ -201,7 +204,10 @@ class ToMany<EntityT> extends Object with ListMixin<EntityT> {
             throw UnimplementedError();
         }
       });
-    });
+      tx.markSuccessful(true);
```

Comment:
Also `if (ownedTx)`? Again, I'm not sure how markSuccessful works with ObjectBox.

Reply:
you can call it multiple times, but yes, no need to do it here if not owned.